.storybook/config.js

```diff
@@ -0,0 +1,11 @@
+const {configure} = require('@kadira/storybook');
+
+require("../data/content/main.css");
+
+function loadStories() {
+  require('../content-test/components/Spotlight.story');
+  require('../content-test/components/ContextMenu.story');
```

Comment:
Nit: odd mix of single and double quotes in this file, which will surely (hopefully) make ESLint cry.